hs-bindgen/src/HsBindgen/SHs/AST.hs

```diff
@@ -142,6 +145,7 @@ data SExpr ctx =
   | EIntegral Integer (Maybe HsPrimType)
   | EFloat Float HsPrimType -- ^ Type annotation to distinguish Float/CFLoat
   | EDouble Double HsPrimType
+  | EString [Word8]
```

Comment:
Can't we use `ByteArray`, or some better representation. `[Word8]` doesn't feel good.